### DISCUSSION TITLE
Calibrating my Vue, please share your thoughts

### DISCUSSION THREAD

**Discussion by eigenphase** [emporia-vue-local/esphome#182] [Ideas]
I am pretty disappointed by the accuracy of the Vue2, especially low powers. Most people don't care but small phantom power adds up and I want it to be accurate even in the single digit watt range. I already calibrated the voltage to what feels sub-1% accuracy.

I have ordered a bunch of 1% {5k, 120, 5} Ohm resistors rated at 5W (was a 70$ purchase at Digikey!) and using series/parallel connections, I created resistances between 144 and 14400 Ohm such that each individual resistor's wattage is below 5W.

With 120V, the values are 0.96W, 2.88W, 5.76W, 8.64W, 20W, 60W, 80W, 99.31W. I am also planning measuring the voltage across the load with a multimeter to correct for the exact wattage. Finally I can use these datapoints to create a (possibly nonlinear) correction in ESPhome.

Since this is already tedious, I am thinking about the easiest way to measure reliably:

1. I plan to run a single wire and put all clamps on the respective phase in at once and read them out in parallel. Any concerns?
2. Can I treat the main phase CT as a normal CT for this purpose?
3. For the purpose of this test, can I just connect both Phase A and Phase B to the same phase and just power it with one phase? Phase shift will be measured as zero then, but I am only interested in making sure that the CTs for each phase give the right wattage. In that case, I could also put all 18(!) CTs across a single wire.
4. I am planning to power the Vue from behind my UPS which should provide a cleaner, more stable voltage. Especially in this case, I would need to connect PhaseA+PhaseB to just the one hot from UPS. Any concerns?

Mainly I want to make sure that the Vue functionally does not rely on some interrelation between Phase CTs and the other CTs and between PhaseA and PhaseB.

Thanks for your thoughts.

**Comment by flaviut** [comment]
1. probably OK, but each clamp does add a load. I don't know what that load would be. My best guess is on the order of 20-30kΩ? I'd do a test.
2. yes, the only difference is the connector size
3. yes, the data is captured for all phase & clamp combinations simultaneously, the config only selects which data to record.
4. sounds reasonable

Sounds like a reasonable procedure, let me know how it works out.

Another thing to consider is temperature--I wonder how much the Vue's readings change depending on the ambient temperature.
